client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -198,10 +198,11 @@ class ThemeEnhancements extends Component {
 						jetpackVersion &&
 						versionCompare( jetpackVersion, '8.1-alpha', '>=' )
 							? translate(
-									'{{b}}Action needed:{{/b}} The Jetpack mobile theme will be retired ' +
-										'and removed from Jetpack in March. Please ensure your current theme ' +
+									'{{b}}Action needed:{{/b}} The Jetpack mobile theme is not supported ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( '{{b}}Action needed:{{/b}} The Jetpack mobile theme will be retired and removed from Jetpack in March. Please ensure your current theme is mobile-ready {{link}}using this tool{{/link}}. If it is not, consider replacing it before March.' )` *ES Score: 6*

Reply:
I must admit I'm still not familiar enough with the bot to always know what to do about its suggestions. Since I'm not going to use the suggestion, I thought I must downrate it?

Reply:
IIRC, the idea with these suggestions is this: it's always a good practice to upvote or downvote every suggestion. We upvote if it's a good suggestion, even if we're not going to use the suggested string. We downvote only if it's a bad suggestion and it doesn't make sense at all. The idea behind this is to gather data to help the bot improve over time. I guess, if not sure what to react with, you can always skip 😉

Reply:
It just tries to suggest strings that might also work and are already translated. Just ignore it if the suggested string is not a match.